### DISCUSSION TITLE
Signup Flow

### DISCUSSION THREAD

**Discussion by jnebab** [web3phl/directory#45] [Brainstorm]
Hey guys, maybe we can start discussing the signup flow if or when we migrate on-chain?

Should we let the signup directly write to the chain? My concern if this is the flow is, that anyone might just spam registration. 

Just want to know your thoughts on this.

**Comment by warengonzaga** [comment]
Good topic to discuss here, as it is on-chain it will be sent as pending for the verifier to check and approach. Something like it will not go through once the sign up has been sent and only will proceed once the team approve it. I think @aindong @alvin-reyes @apquinit @kquirapas can explain more.

We should have an opensource meeting via gather next time.

Connecting this here #24
